nemo/core/config/hydra_runner.py

```diff
@@ -51,14 +46,42 @@ def wrapper(cfg_passthrough: Optional[DictConfig] = None) -> Any:
                 args = get_args_parser()
 
                 # Parse arguments in order to retrieve overrides
-                parsed_args = args.parse_args()
+                parsed_args = args.parse_args()  # type: argparse.Namespace
 
                 # Get overriding args in dot string format
                 overrides = parsed_args.overrides  # type: list
 
-                # Update overrides
+                # Disable the creation of .hydra subdir
+                # https://hydra.cc/docs/tutorials/basic/running_your_app/working_directory
+                overrides.append("hydra.output_subdir=null")
+                # Hydra logging outputs only to stdout (no log file).
+                # https://hydra.cc/docs/configure_hydra/logging
+                overrides.append("hydra/job_logging=stdout")
```

Comment:
So long as this stops hydra from interfering with the root logger, I am happy with this

Reply:
Resolved after discussion